packages/react-ui-ag/src/Banners/index.js

```diff
@@ -0,0 +1 @@
+export { default as Banner } from './Banner'
```

Comment:
Please follow the same syntax as the other components https://github.com/rentpath/react-ui/blob/master/packages/react-ui-core/src/Button/index.js